core/src/main/java/bisq/core/offer/TriggerPriceService.java

```diff
@@ -152,6 +156,20 @@ private void checkPriceThreshold(MarketPrice marketPrice, OpenOffer openOffer) {
             openOfferManager.deactivateOpenOffer(openOffer, () -> {
             }, errorMessage -> {
             });
+        } else if (openOffer.getState() == OpenOffer.State.AVAILABLE) {
+            // check the mempool if it has not been done before
+            if (openOffer.getMempoolStatus() < 0 && mempoolService.canRequestBeMade()) {
+                mempoolService.validateOfferMakerTx(openOffer.getOffer().getOfferPayload(), (txValidator -> {
+                    openOffer.setMempoolStatus(txValidator.isFail() ? 0 : 1);
```

Comment:
So if it fails to validate the tx the offer will be deactivated after the next price update (~60 sec later), right?

Reply:
Yes exactly.  Cleaner than duplicating the deactivation code within the async response.

One thing I'd like to do though is delay 60 seconds before checking a new offer, because at the moment it checks immediately upon creation which can be too soon for the tx to have propagated.